shiny/ui/_include_helpers.py

```diff
@@ -0,0 +1,300 @@
+    document. If instead, you want to place the tag in the :func:`~ui.tags.head` of the
+    document, you can wrap it in ``head_content`` (in this case, just make sure you're
+    aware that the DOM probably won't be ready when the script is executed).
+
+    .. code-block:: python
+
+    from htmltools import head_content from shiny import ui
+
+    ui.fluidPage(
+        head_content(ui.include_javascript("custom.js"))
+    )
+
+    See Also
+    --------
+    ~ui.tags.script
+    ~include_css
+    ~include_html
+    """
+
+    if method == "inline":
+        return tags.script(read_utf8(path), type="text/javascript")
+
+    include_files = method == "link_files"
+    path_dest, hash = maybe_copy_files(path, include_files)
+
+    dep, src = create_include_dependency("include-js-" + hash, path_dest, include_files)
+
+    return tags.script(dep, src=src)
+
+
+@add_example()
+def include_css(
+    path: str, *, method: Literal["link", "link_files", "inline"] = "link"
+) -> Tag:
+    """
+    Include a CSS file
+
+    Parameters
+    ----------
+    path
+        A path to a CSS file.
+    method
+        One of the following:
+          * ``"link"``: Link to the CSS file via a :func:`~ui.tags.link` tag. This
+            method is generally preferrable to ``"inline"`` since it allows the browser
+            to cache the file.
+          * ``"link_files"``: Same as ``"link"``, but also allow for the CSS file to
+            request other files within ``path``'s immediate parent directory (e.g.,
+            ``@import()`` another file). Note that this isn't the default behavior
+            because you should **be careful not to include files in the same directory
+            as ``path`` that contain sensitive information**. A good general rule of
+            thumb to follow is to have ``path`` be located in a subdirectory of the app
+            directory. For example, if the app's source is located at ``/app/app.py``,
+            then ``path`` should be somewhere like ``/app/css/custom.css`` (and all the
+            other relevant accompanying 'safe' files should be located under
+            ``/app/css/``).
+          * ``"inline"``: Inline the CSS file contents within a :func:`~ui.tags.style`
+            tag.
+
+    Returns
+    -------
+    If ``method="inline"``, returns a :func:`~ui.tags.style` tag; otherwise, returns a
+    :func:`~ui.tags.link` tag.
+
+    Note
+    ----
+    By default this places a :func:`~ui.tags.link` (or :func:`~ui.tags.style`) tag in
+    the :func:`~ui.tags.body` of the document, which isn't optimal for performance, and
+    may result in a Flash of Unstyled Content (FOUC). To instead place the CSS in the
+    :func:`~ui.tags.head` of the document, you can wrap it in ``head_content``:
+
+    .. code-block:: python
+
+    from htmltools import head_content from shiny import ui
+
+    ui.fluidPage(
+        head_content(ui.include_css("custom.css"))
+    )
```

Comment:
It would be very helpful if there was also an example that showed how to inline css (with something like `head_content(ui.tags.style(" .... "))`, because many users will see `include_css` in the API docs and will be looking for that.